msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java

```diff
@@ -517,10 +518,10 @@ public void acquireTokenSilentAsync(@NonNull final String[] scopes,
     @WorkerThread
     public IAuthenticationResult acquireTokenSilent(@NonNull final String[] scopes,
                                                     @NonNull final String authority) throws MsalException, InterruptedException {
-
         final IAccount persistedAccount = getPersistedCurrentAccount();
         if (persistedAccount == null) {
-            throw new MsalClientException(MsalClientException.NO_CURRENT_ACCOUNT);
+            MsalClientException exception = new MsalClientException(MsalClientException.NO_CURRENT_ACCOUNT);
```

Comment:
why a new variable ?

Reply:
I don't remember doing it. I might have accidentally clicked on the `introduce local variable` suggestion within Android Studio and that might have done it. I'll remove it.

Reply:
@kreedula Addressed